libraries/AP_AHRS/AP_AHRS_DCM.cpp

```diff
@@ -994,6 +996,17 @@ void AP_AHRS_DCM::estimate_wind(void)
         return;
     }
 
+#if AP_AIRSPEED_ENABLED
```

Comment:
Uh - shouldn't this go below the `diff_length >0.2f` case?!

Reply:
Yes..... My head mixed up which if should be the early return.